netbox/project-static/src/sidenav.ts

```diff
@@ -27,6 +27,12 @@ class SideNav {
    */
   private sections: Section[] = [];
 
+  /**
+   * Tracker if the mouse on the navbar or not
+   */
+  // private mouseOver: boolean = false;
+
+
```

Comment:
Should be free to remove all these commented-out script lines.

Reply:
Yep.. I left them in, just in case the CSS was not doing it as we want